Formula/pyside.rb

```diff
@@ -28,14 +30,23 @@ class Pyside < Formula
   uses_from_macos "libxml2"
   uses_from_macos "libxslt"
 
+  resource "packaging" do
+    url "https://files.pythonhosted.org/packages/df/9e/d1a7217f69310c1db8fdf8ab396229f55a699ce34a203691794c5d1cad0c/packaging-21.3.tar.gz"
+    sha256 "dd47c42927d89ab911e606518907cc2d3a1f38bbd026385970643f9c5b8ecfeb"
+  end
+
   def install
     # upstream issue: https://bugreports.qt.io/browse/PYSIDE-1684
     inreplace "sources/pyside6/cmake/Macros/PySideModules.cmake",
               "${shiboken_include_dirs}",
               "${shiboken_include_dirs}:#{Formula["qt"].opt_include}"
 
+    python = Formula["python@3.9"]
+    venv = virtualenv_create(buildpath/"venv", python.opt_bin/"python3")
```

Comment:
```suggestion
    venv = virtualenv_create(buildpath/"venv", "python3")
```

Reply:
Looks like this was needed. That's unexpected. Apologies for the bad suggestion!

Reply:
Never mind.😁